apps/requestron/Jenkinsfile

```diff
@@ -0,0 +1,66 @@
+pipeline {
+    agent none
+    options {
+        disableResume()
+    }
+    stages {
+        stage('Build') {
+            agent { label 'build' }
+            steps {
+                script {
+                    def filesInThisCommitAsString = sh(script:"git diff --name-only HEAD~1..HEAD | grep '^apps/requestron/' | grep -v -e 'apps/requestron/README.md' -e 'apps/requestron/Jenkinsfile' || echo -n ''", returnStatus: false, returnStdout: true).trim()
+                    def hasChangesInPath = (filesInThisCommitAsString.length() > 0)
+                    echo "${filesInThisCommitAsString}"
+                    if (!currentBuild.rawBuild.getCauses()[0].toString().contains('UserIdCause') && !hasChangesInPath){
+                        currentBuild.rawBuild.delete()
+                        echo "No changes detected in the path (apps/requestron/ other than Jenkinsfile and docs)"
+                        return
+                    }
+                }
+                echo "Aborting all running jobs ..."
+                script {
+                    abortAllPreviousBuildInProgress(currentBuild)
+                }
+                echo "Building ..."
+                sh "cd apps/requestron/.pipeline && ./npmw ci && ./npmw run build -- --pr=${CHANGE_ID}"
+            }
+        }
+        // stage('Deploy (DEV)') {
+        //     agent { label 'deploy' }
+        //     steps {
+        //         echo "Deploying ..."
+        //         sh "cd apps/documize/.pipeline && ./npmw ci && ./npmw run deploy -- --pr=${CHANGE_ID} --env=dev"
+        //     }
+        // }
+        stage('Deploy (TEST)') {
+            agent { label 'deploy' }
+//             when {
+//                 expression { return env.CHANGE_TARGET == 'master';}
+//                 beforeInput true
+//             }
+            input {
+                message "Should we continue with deployment to TEST?"
+                ok "Yes!"
+            }
+            steps {
+                echo "Deploying ..."
+                sh "cd apps/requestron/.pipeline && ./npmw ci && ./npmw run deploy -- --pr=${CHANGE_ID} --env=test"
+            }
+        }
+        stage('Deploy (PROD)') {
+            agent { label 'deploy' }
+            when {
```

Comment:
we probably don't need this as we are not using a release branch. By default, all pr should be deployable to production.